packages/playwright/src/index.ts

```diff
@@ -169,12 +183,12 @@ export default class AxeBuilder {
 
   private script(): string {
     return `
-        ${this.source}
-        axe.configure({ 
-          allowedOrigins: ['<unsafe_all_origins>'], 
-          branding: { application: 'playwright' }
-        })
-        `;
+      ${this.source}
+      axe.configure({
+        ${this.legacyMode ? '' : 'allowedOrigins: ["<unsafe_all_origins>"],'}
```

Comment:
```suggestion
        ${this.legacyMode ? ''allowedOrigins: ["<same_origin>"]' : 'allowedOrigins: ["<unsafe_all_origins>"],'}
```

Reply:
That's exactly the same thing but with more characters.